packages/lesmis-server/src/app/createApp.ts

```diff
@@ -41,8 +41,8 @@ export function createApp() {
     .get('/v1/map-overlays/:entityCode', handleWith(MapOverlaysRoute))
     .get('/v1/entity/:entityCode', handleWith(EntityRoute))
     .get('/v1/report/:entityCode/:reportCode', handleWith(ReportRoute))
+    .put('/v1/updateSurveyResponse/:id', handleWith(UpdateSurveyResponseRoute))
```

Comment:
Sorry I realise I suggested the name, but have just realised that the http verb `put` implies the `update`, so we can just call this `/v1/surveyResponse/:id`. All the other `Update` places are appropriate, just the endpoint should change 🙏

Reply:
Good point. Should it be snake case too?

Reply:
Oh yeah looks like it based on the other endpoints in here, didn't even notice that 👀